src/ontology/uberon-edit.obo

```diff
@@ -44033,7 +44033,7 @@ xref: UMLS:C0006270 {source="ncithesaurus:Bronchiole"}
 xref: VHOG:0000675
 xref: Wikipedia:Bronchiole
 is_a: UBERON:0000117 ! respiratory tube
-relationship: branching_part_of UBERON:0002185 ! bronchus
+relationship: branching_part_of UBERON:0007196 ! tracheobronchial tree
 relationship: part_of UBERON:0002048 {source="MA"} ! lung
 relationship: part_of UBERON:0007196 {source="FMA"} ! tracheobronchial tree
 relationship: part_of UBERON:0010368 {source="ncit"} ! pulmonary lobule
```

Comment:
`branching part of` and `part of` isn't redundant for this term?

Reply:
You are right, I will remove the relationship

Reply:
the `part of` axiom was not inferred after I deleted it.
Could benefit from @dosumis's feedback...